analytics/index.yaml

```diff
@@ -0,0 +1,727 @@
+        - name: e_n
+          in: formData
+          description: Custom event name.
+          type: string
+          minLength: 1
+          #TODO: Specify maximal allowed length.
+          example: finantial-report
+        - name: e_v
+          in: formData
+          description: Custom event value.
+          type: number
+          format: float
+          minimum: 0
+          example: 201704
+        #TODO: Link to content tracking description.
+        - name: c_n
+          in: formData
+          description: Content name.
+          type: string
+          minLength: 1
+          #TODO: Specify maximal allowed length.
+          example: recommended-products
+        - name: c_p
+          in: formData
+          description: Content piece.
+          type: string
+          minLength: 1
+          #TODO: Specify maximal allowed length.
+          example: Unicorn Iron on Patch
+        - name: c_t
+          in: formData
+          description: Content target.
+          type: string
+          minLength: 1
+          #TODO: Specify maximal allowed length.
+          example: http://example.com/product/unicorn-iron-on-path.html
+        - name: c_i
+          in: formData
+          description: Content interaction.
+          type: string
+          minLength: 1
+          #TODO: Specify maximal allowed length.
+          example: show details
+        #TODO: Decide if media analytics should be described in separate document and link to it if yes. For now those parameters are not included in this specification.
+        - name: send_image
+          in: formData
+          description: |
+            <p>Request response content.</p>
+            <p>"0" - HTTP 204 [No content].<br>"1" - HTTP 200 [1 pixel GIF].</p>
+          type: integer
+          minimum: 0
+          maximum: 1
+          default: 1
+          example: 0
+        - name: bots
+          in: formData
+          description: |
+            <p>Bots tracking.</p>
+            <p>"0" - disabled.<br>"1" - enabled.</p>
+          type: integer
+          minimum: 0
+          maximum: 1
+          default: 0
+          example: 0
+        - name: token_auth
+          in: formData
+          description: |
+            <p>Authentication token.</p>
+            <p>Override parameters are usually used for debugging. They should be used only by authorized personel
+            (SuperUser or Admin) and requires "token_auth".</p>
+          type: string
+          minLength: 32
+          maxLength: 32
+          example: caa18074df8347b684d7b943b2484fe9
+        - name: cip
+          in: formData
+          description: |
+            <p>Override IP.</p>
+            <p>Override parameters are usually used for debugging. They should be used only by authorized personel
+            (SuperUser or Admin) and requires "token_auth".</p>
+          type: string
+          format: IPv4|IPv6
+          example: 172.16.254.1
+        - name: cdt
```

Comment:
Usually used in log importing, requires token auth if visit time is different from server time more then 30min.

Reply:
Dates in the future are ignored and set to `now`